### PR TITLE
Update flow on creating tokens for registry.redhat.io tokens

### PR DESCRIPTION
**Pull request by arewm**
The process was not clearly documented after going through it. Proposing changes based on experience.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc [hunk 1]
```diff
@@ -240,15 +240,15 @@ Create the Red Hat Container Registry token by following these steps:
...
 .Procedure  
 
 . Go to *Component Settings*. 
 . Select *Create new build secret*.
 . In the *Create new build secret* modal, add the *registry.redhat.io* token that you want to use as a pull secret:
 .. In the *Key* field, enter *.dockerconfigjson*.
-.. In the *Select or Enter Name* field, enter *registry-redhat.io*. 
+.. In the *Select or Enter Name* field, enter *registry-redhat-io*. 
 .. Enter the registry service account token and click *Create*.
 .. From your  command line, run the `oc secrets link appstudio-pipeline registry-redhat-io` command.
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:253]: This line isn't clear to me either. You are in the RHTAP GUI and then you just tell people to go to the command line. Is there any documentation on configuring this that we can link to in order to help with the transition?

Comment (jc-berger) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:255]: We do not have any particular documentation on configuring this type of CLI work to help with the transition. 

We thought that providing some CLI instruction would help. I really like your additions, and I think that will help provide clarity and smooth out the transition from UI to CLI here.

**Comment by jc-berger** [comment]
I've opened the following PR to address the concerns raised here: https://github.com/redhat-appstudio/docs.appstudio.io/pull/131

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc [hunk 2]
```diff
@@ -226,8 +226,8 @@ Create the Red Hat Container Registry token by following these steps:
 . Create a registry service account by clicking *New Service Account*.
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:226]: Create a registry service account by clicking *New Service Account*. You are taken to the page, *Create a New Registry Service Account*.

Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:226]: Create a registry service account by clicking *New Service Account*. You are taken to the page, **Create a New Registry Service Account**.

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc [hunk 3]
```diff
@@ -226,8 +226,8 @@ Create the Red Hat Container Registry token by following these steps:
 . Create a registry service account by clicking *New Service Account*. 
 . Fill in the *Name* and *Description* field. 
 . Click *Create*. 
-. For *Token Information*, click *Docker Configuration*. 
-. Enter the following Dockerfile registry authentication code:
+. On the *Token Information* page, click *Docker Configuration* and download or *view its contents* to display the authentication.
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:229]: . Click **Create**. You are taken to the page, *Token Information*. 
. Click **Docker Configuration**. 
. To display the Dockerfile registry authentication code, download the authentication of your new account or click **view its contents**. 
. Enter the Dockerfile registry authentication code. It should look similar to this example:

**Review by jc-berger — COMMENTED** [review]
see some minor doc tweaks. otherwise, changes look great :)

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc (hunk 3 above)
Comment (arewm) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:229]: The registry authentication is not a code, so I don't think that part of the change makes sense. The registration is a json object containing a base64 encoded string.

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc [hunk 4]
```diff
@@ -240,16 +240,18 @@ Create the Red Hat Container Registry token by following these steps:
 
 === Configuring your application to use a Red Hat Container Registry token 
 
-After creating the Red Hat Container Registry token, include the token in your application. When your application includes the Red Hat Container Registry token, you can access *registry.redhat.io* which improves your automation. 
+After creating and saving the Red Hat Container Registry token, you now need to configure it in your application's builds. When your workspace includes the Red Hat Container Registry token, you can access *registry.redhat.io* container images, enabling you to build off of trusted base images. 
 
 .Procedure  
 
-. Go to *Component Settings*. 
+. In the {ProductName} UI, go to your application components and click on *Edit component settings* in the kebab menu.
```
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc:247]: If you do not direct users to the kebab menu, they will not be able to find the `edit component settings` to click on. I think it still makes sense to include that detail.

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc (hunk 3 above)
Comment (jc-berger) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc]: Okay, then would the following revision work?:

To display the Dockerfile registry authentication, download the authentication of your new account or click view its contents.
Enter the Dockerfile registry authentication. It should look similar to this example:

So, essentially, simply removing the word `code`

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (jc-berger) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc]: I hear you about it making sense, but Red Hat does prohibit us from using words like `kebab` and menu` . They are not considered user friendly or accessible, so Red Hat doesn't allow it. 

We can fiddle around with the sentence structure more, but in my experience, we tend to just omit phrases like "in the kebab menu"

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc [hunk 5]
```diff
@@ -240,16 +240,18 @@ Create the Red Hat Container Registry token by following these steps:
...
 
-. Go to *Component Settings*. 
+. In the {ProductName} UI, go to your application components and click on *Edit component settings* in the kebab menu. 
 . Select *Create new build secret*.
 . In the *Create new build secret* modal, add the *registry.redhat.io* token that you want to use as a pull secret:
 .. In the *Key* field, enter *.dockerconfigjson*.
-.. In the *Select or Enter Name* field, enter *registry-redhat.io*. 
+.. In the *Select or Enter Name* field, enter *registry-redhat-io*. 
 .. Enter the registry service account token and click *Create*.
+... If you have not previously created a secret with this name then it should be successfully created. If you have already created a secret with the *registry-redhat-io* name, however, you will need to delete the current secret before being able to re-create it.
```
[the remark this answers is not in the document]
Comment (jc-berger) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc]: If you want to make a deeper sub-step by using `...` we can. However, for readability and accessibility, we tend to want to keep it to a series of one sub steps, so `..` 

If you, as a developer, don't think users would get thrown off, we can document it as you initially had it.

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc]: I hear what you are saying about the Red Hat style guides. How can we help discovery of the menu and its inclusion of `edit component settings`? As a user coming to UI and trying to find this setting, I would get lost if I hadn't done it before.

**Pull request closed without merging**